ntbs-service/Pages/Help/Menu.cshtml

```diff
@@ -18,4 +24,9 @@
             Service directory
         </a>
     </div>
-</nhs-width-container>
\ No newline at end of file
+    <div>
+        <a class="submenu-list-item-non-js" href="@links.SharePointHomePage" target="_blank" rel="noopener noreferrer">
+            NTBS Resources
```

Comment:
In general it's a bit weird to refer to your own site by name (i.e. if the user just saw "Resources" here, they would assume they are NTBS resources because they're already on the NTBS site). So maybe "Resources" or "User resources" would be a better name?

Reply:
Yeah I agree, @aptemus what do you think? I think it originally said "NTBS Resources" on the ticket

Reply:
Yes, it does look a bit odd.  I think I used that name because that's the actual name of the SharePoint site (I was tempted to just put 'SharePoint') but Resources probably makes more sense.

Reply:
Great, thanks both!